client/src/components/content/BpmnRobotModeler/BpmnRobotModeler.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import BpmnModeler from 'bpmn-js/lib/Modeler';
+import { emptyBpmn } from '../../../resources/modeler/empty.bpmn';
+import propertiesProviderModule from 'bpmn-js-properties-panel/lib/provider/camunda';
+import camundaModdleDescriptor from 'camunda-bpmn-moddle/resources/camunda';
+import arkRPA_ModdleDescriptor from '../../../resources/modeler/modelerPropertiesExtensionRPA/ark-rpa';
+import parseDiagramJson from '../../../utils/parser.js';
+import convert from 'xml-js';
+import downloadString from '../../../utils/downloadString.js';
+import 'bpmn-js/dist/assets/diagram-js.css';
+import 'bpmn-font/dist/css/bpmn-embedded.css';
+import ModelerSidebar from '../ModelerSidebar/ModelerSidebar';
+import styles from './BpmnRobotModeler.module.css';
+
+import { Layout } from 'antd';
+
+const { Content } = Layout;
+
+/**
+ * @component
+ * @description This component renders the modeling interface as well as the sidebar.
+ */
+const BpmnRobotModeler = () => {
```

Comment:
Not really happy with the naming here. All we do is model robots, so why does that need to explicitly be in the name? I would prefer the naming to just reflect, what kind of modeling interface this is, not really what it models

Reply:
Good point. I will change that.